components/Heroes/heroes.css

```diff
@@ -0,0 +1,79 @@
+.Header {
```

Comment:
I think it's important to make convention for naming css files and components, because for this particular problem current css files have different naming convention.